packages/swapper/src/swappers/ZrxSwapper/endpoints.ts

```diff
@@ -34,7 +34,7 @@ export const zrxApi: SwapperApi = {
     // TODO(gomes): when we wire this up, this should consume getZrTradeQuote and we should ditch this guy
     // getTradeQuote() is currently consumed at input time (for all swappers, not just ZRX) with weird Frankenstein "quote endpoint fetching ZRX rate endpoint
     // but actually expecting quote input/output" logic. This is a temporary method to get the ZRX swapper working with the new swapper architecture.
-    const tradeQuoteResult = await getZrxPseudoTradeQuote(
```

Comment:
This means we will *have* to enable the flag as part of this PR or else things will break

Reply:
Yeah unfortunately flag *has* to go in and can't really keep the flag off with this change, so we will need all the extra testing caution here 🙏🏽